### DISCUSSION TITLE
Support for pypi install ?

### DISCUSSION THREAD

**Discussion by gmaze** [Parcels-code/Parcels#1162] [General]
Hi,
I understand from #1134 that parcels can't be installed with pip at this point
I think this is a strong limitation to the development of third party packages depending on Parcels if these downstream packages are using pypi to define requirements, rather than conda. 
Many packages support both, and very often conda is simply a wrapper around a pypi setup and distribution

Would you consider adding support for pypi install ? (so that third party packages can decide which one they want to use)

**Comment by erikvansebille** [comment]
Thanks for this idea, I see your point about also supporting pip for dependencies. However, I have no experience at all with setting up pip; would you be able to help us?

**Reply by gmaze** [reply]
Yes, I give it a try in a new PR
